test-cases/longevity/longevity-ics-nemesis-alter-table-1TB-7days.yaml

```diff
@@ -0,0 +1,37 @@
+
+stress_cmd: ["cassandra-stress mixed         cl=QUORUM duration=10080m -schema 'replication(factor=3)                               compaction(strategy=LeveledCompactionStrategy)'    -port jmx=6868 -mode cql3 native  -rate threads=20 -pop seq=1..1100200300  -log interval=5 -col 'size=FIXED(1024) n=FIXED(1)'",
+             "cassandra-stress write         cl=QUORUM duration=10010m -schema 'replication(factor=3) compression=LZ4Compressor     compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=lz4                   -rate threads=50 -pop seq=1..50000000    -log interval=5",
+             "cassandra-stress write         cl=QUORUM duration=10020m -schema 'replication(factor=3) compression=SnappyCompressor  compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=snappy                -rate threads=50 -pop seq=1..50000000    -log interval=5",
+             "cassandra-stress write         cl=QUORUM duration=10030m -schema 'replication(factor=3) compression=DeflateCompressor compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=none                  -rate threads=50 -pop seq=1..50000000    -log interval=5",
+             ]
+
+stress_read_cmd: ["cassandra-stress read cl=QUORUM duration=10080m                                                                                                                 -port jmx=6868 -mode cql3 native  -rate threads=10 -pop seq=1..1100200300  -log interval=5 -col 'size=FIXED(1024) n=FIXED(1)'",
+                  "cassandra-stress read cl=QUORUM duration=10010m -schema 'replication(factor=3) compression=LZ4Compressor compaction(strategy=SizeTieredCompactionStrategy)'     -port jmx=6868 -mode cql3 native compression=lz4                   -rate threads=20 -pop seq=1..50000000    -log interval=5",
+                  "cassandra-stress read cl=QUORUM duration=10020m -schema 'replication(factor=3) compression=SnappyCompressor compaction(strategy=SizeTieredCompactionStrategy)'  -port jmx=6868 -mode cql3 native compression=snappy                -rate threads=20 -pop seq=1..50000000    -log interval=5",
+                  "cassandra-stress read cl=QUORUM duration=10030m -schema 'replication(factor=3) compression=DeflateCompressor compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=none                  -rate threads=20 -pop seq=1..50000000    -log interval=5",
+                  ]
+run_fullscan: 'random, 17'  # 'ks.cf|random, interval(min)''
+
+round_robin: true
+
+n_db_nodes: 4
+n_loaders: 2
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.4xlarge'
+instance_type_loader: 'c5.2xlarge'
+
+nemesis_class_name: 'ToggleTableIcsMonkey'
+nemesis_interval: 5
+
+user_prefix: 'longevity-ics-nemesis-1tb-7d'
```

Comment:
change to longevity-ics-toggle-strategy-500gb-24h

Reply:
done